utils/install-gatling.sh

```diff
@@ -2,4 +2,4 @@
 
 curl -sS https://repo1.maven.org/maven2/io/gatling/highcharts/gatling-charts-highcharts-bundle/2.3.1/gatling-charts-highcharts-bundle-2.3.1-bundle.zip > gatling.zip
```

Comment:
it would be cool to have instructions to use gatling 3 instead of 2, but it can be done in another PR

Reply:
It will be done :)